code/game/gamemodes/cult/cult.dm

```diff
@@ -37,10 +37,10 @@
 	false_report_weight = 10
 	restricted_jobs = list("Chaplain","AI", "Cyborg", "Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Head of Personnel")
 	protected_jobs = list()
-	required_players = 24
-	required_enemies = 4
+	required_players = 20
+	required_enemies = 2
```

Comment:
Did we forget that Cult requires 3 people or was that changed at some point, just fyi